Tasks/Common/VstsAzureHelpers_/ImportFunctions.ps1

```diff
@@ -3,35 +3,48 @@
     param(
         [Parameter(Mandatory = $true)]
         [ValidateSet('Azure', 'AzureRM')]
-        [string[]]$PreferredModule)
+        [string[]]$PreferredModule,
```

Comment:
Keep the formatting consistent for all arguments. Have space between the type and variable. I see this in other functions as well. Please correct in all instances.